app/context.go

```diff
@@ -248,6 +248,7 @@ type StorageCtx struct {
 	Balances   *balance.Store
 	Domains    *ons.DomainStore
 	Validators *identity.ValidatorStore // Set of validators currently active
+	Witnesses  *identity.WitnessStore
```

Comment:
We need to add witnesses to the Job context as well  ?
We can rename the JobContext as Ethereum and Bitcoin job context ,and separate out the events package ?